2404.js

```diff
@@ -0,0 +1,28 @@
+// △
+/**
+ * @param {number[]} nums
+ * @return {number}
+ */
+var mostFrequentEven = function (nums) {
+  let evenList = [];
```

Comment:
You can extract even numbers in nums by using Array.filter(), so that you don't need to check if numbers are even or not in the loop.

`let evenList = nums.filter((num) => num % 2 === 0);`